tern/analyze/docker/analyze.py

```diff
@@ -106,7 +106,12 @@ def analyze_subsequent_layers(image_obj, shell, master_list, redo, dfobj=None,
                               dfile_lock=False):
     # get packages for subsequent layers
     curr_layer = 1
+    #set current work_dir to none
+    work_dir = None
     while curr_layer < len(image_obj.layers):  # pylint:disable=too-many-nested-blocks
+        #update workdir
```

Comment:
small nit: should be a space between `#` and comment. Could you also update this comment with more context? i.e. `# If workdir changes, update value accordingly so we can later execute base.yml commands from the workdir.`